### ISSUE TITLE
Stack

### ISSUE THREAD

**Issue by teenie-quaggard** [founders-programme-2/business-launchpad#21]
One thing to discuss will be stack.

After loosely creating the Figma, I propose:

React for the front,
Express / Node server,
MongoDb for our database

Potentially airtable to store some fields that could be updated by Business Launchpad later:
-Resources
-Goal categories
-Potentially highlight color variable

and some sort of authentication (OAuth?). I am watching this tonight, so maybe I'll have some ideas...
https://www.youtube.com/watch?v=s1swJLYxLAA

**Comment by SalwaMugh** [comment]
I'm good for the stack , there is a challenge in it and I like that!
what do you both think @Oliversw  @shaima96 ??

**Comment by shaima96** [comment]
For me, it's ok.
But if we can use Sequelize instead of MongoDB??
Because I have knowledge of it more than MongoDB.
If you don't want it is ok I can learn it.

**Comment by teenie-quaggard** [comment]
@shaima96 I am quite keen to try MongoDb as it is so widely used. I've noticed that a lot of employers ask for knowledge of it. Neither Oliver or I have ever used it, so we will be on the same page!
That said, I've never used Sequelize either, so I am down for whatever!

@SalwaMugh & @Oliversw - do either of you have a preference?

**Comment by SalwaMugh** [comment]
I've never used neither so I'm also open for it, also for the reason you mentioned it, it's more preferable to use another type of database! Mongo.

**Comment by teenie-quaggard** [comment]
An aside, but could be helpful later on:
Here is a Todo App TDD example using mongodb :)

https://semaphoreci.com/community/tutorials/a-tdd-approach-to-building-a-todo-api-using-node-js-and-mongodb

**Comment by Oliversw** [comment]
I agree with @teenie-quaggard and think that it would be nice to have some experience with MongoDB! :)
